docs/source/releases.rst

```diff
@@ -101,7 +101,7 @@ If you were previously using https://github.com/ioos/qartod , see the *Migrating
     * Split out :code:`threshold` parameters into explicit :code:`suspect` and :code:`fail` parameters (`#4 <https://github.com/ioos/ioos_qc/pull/4>`_)
     * Bug fixes
 * Remove support for python 2
-* Upgrade numpy to 1.14
+* Upgrade numpy to < 2.0.0
```

Comment:
You don't edit past release notes. That is still true for the 0.1.0 from Feb 2019.